### PR TITLE
Add guidelines for promoting Committers to PMC

### PR DESCRIPTION
**Pull request by kaxil**
This is based on the discussion on the mailing list and proposal from various PMC Members

---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -135,6 +135,67 @@ The key aspects of a committer are:
...
+---------------------------------------------------
+
+To become a PMC member the committers should meet all **general prerequisites**.
+Apart from that the person should demonstrate distinct **community involvement** or **code contributions**.
+
+Prerequisites
+==============
+
+* Has been a committer for at least 3 months
+* Is still active community member
```
Comment (turbaszek) [inline CONTRIBUTING.rst]: What is the measure of activity?

File: CONTRIBUTING.rst [hunk 2]
```diff
@@ -135,6 +135,67 @@ The key aspects of a committer are:
...
+
+Community involvement
+======================
+
+* Visibility on discussions on the dev mailing list
+* Spreading the “Airflow” either:
+
+  * Talks at meetups, conferences, etc
+  * Creating content (videos, blogs etc)
```
Comment (turbaszek) [inline CONTRIBUTING.rst]:
```suggestion
  * Creating content like videos, blogs, etc
```
Will be more consistent with previous bullet

File: CONTRIBUTING.rst [hunk 3]
```diff
@@ -135,6 +135,67 @@ The key aspects of a committer are:
 * Visibility in community discussions (dev mailing list, Slack and GitHub)
 * Testing Release Candidates
 
+Guidelines for promoting Committers to Airflow PMC
```
Comment (turbaszek) [inline CONTRIBUTING.rst:138]: Should we link somewhere the official ASF guidelines?

Comment (kaxil) [reply CONTRIBUTING.rst:138]: Updated in https://github.com/apache/airflow/pull/13090/commits/4bc8e4ef24368afc778f2dbfd2bc4056c84add8e

File: CONTRIBUTING.rst (hunk 2 above)
Comment (kaxil) [reply CONTRIBUTING.rst]: Updated in https://github.com/apache/airflow/pull/13090/commits/4bc8e4ef24368afc778f2dbfd2bc4056c84add8e

File: CONTRIBUTING.rst (hunk 1 above)
Comment (kaxil) [reply CONTRIBUTING.rst]: Updated in https://github.com/apache/airflow/pull/13090/commits/4bc8e4ef24368afc778f2dbfd2bc4056c84add8e

**Review by turbaszek — APPROVED** [review]

**Pull request merged**